apps/registration/src/assets/css/index.css

```diff
@@ -1 +1,8 @@
 @import "tailwind/custom";
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+Thai&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Baloo+2&display=swap');
+
+body {
+  background-color: #6C30A4;
+  font-family: 'Baloo 2', 'Noto Sans Thai';
```

Comment:
@Qandle Please move font config to individual tailwind config where you want to set the font (dashboard / registration)